core/templates/dev/head/pages/exploration_editor/translation_tab/state_translation_status_graph_directive.html

```diff
@@ -4,7 +4,7 @@
 
 <md-card style="background-color: white; margin: 20px 0px; padding: 8px;">
   <div class="oppia-translation-state-graph-container protractor-test-translation-graph">
-    <div state-graph-viz graph-data="getGraphData()" current-state-id="getActiveStateName()" on-click-function="onClickStateInMap" allow-panning="true" center-at-current-state="true" style="height: 100%;">
+    <div state-graph-viz graph-data="getGraphData()" current-state-id="getActiveStateName()" on-click-function="onClickStateInMap" allow-panning="true" center-at-current-state="true" node-colors="nodeColors()" show-needs-update-sign="true" style="height: 100%;">
```

Comment:
should show-needs-update-sign be explicitly set to false on the other state-graph-viz directives?

Also, I would consider just calling this `show-translation-warnings-only`, as it's a little more clear to me (since show-needs-update-sign being set to true doesn't imply that other warnings AREN'T shown), but it's up to you.

Reply:
> should show-needs-update-sign be explicitly set to false on the other state-graph-viz directives?

Nope! It will only show when it's provided!

Alos, changing it to show-translation-warnings.